src/GlobalStyles.js

```diff
@@ -11,6 +11,15 @@ export const GlobalStyles = createGlobalStyle`
     box-sizing: inherit;
   }
 
+  body {
```

Comment:
I think it might be more optimal to style either body or #root instead of applying the same styles for both.

Reply:
I remove well